examples/incompressible_flow/2d_transient_flow_around_cylinder/cylinder.prm

```diff
@@ -0,0 +1,148 @@
+end
+
+#---------------------------------------------------
+# Force
+#---------------------------------------------------
+subsection forces
+    set verbosity             = verbose
+    set calculate force       = true
+    set calculate torque      = false
+    set force name            = force
+    set output precision      = 10
+    set calculation frequency = 1
+    set output frequency      = 1
+end
+
+#---------------------------------------------------
+# Initial condition
+#---------------------------------------------------
+subsection initial conditions
+    set type = nodal
+    subsection uvwp
+            set Function expression = 1; 0; 0
+    end
+end
+
+#---------------------------------------------------
+# Physical Properties
+#---------------------------------------------------
+subsection physical properties
+  subsection fluid 0
+    set kinematic viscosity            = 0.005
+  end
+end
+
+#---------------------------------------------------
+# Mesh
+#---------------------------------------------------
+subsection mesh
+    set type                 = gmsh
+    set file name            = cylinder_structured.msh
+    set initial refinement   = 1
+end
+
+# --------------------------------------------------
+# Boundary Conditions
+#---------------------------------------------------
+subsection boundary conditions
+  set number                  = 3
+    subsection bc 0
+        set type              = noslip
+    end
+    subsection bc 1
+        set type              = function
+        subsection u
+            set Function expression = 1
+        end
+        subsection v
+            set Function expression = 0
+        end
+        subsection w
+            set Function expression = 0
+        end
+    end
+    subsection bc 2
+        set type              = slip
+    end
+end
+
+# --------------------------------------------------
+# Manifolds
+#---------------------------------------------------
+subsection manifolds
+  set number                  = 1
+    subsection manifold 0
+        set id                = 0
+        set type              = spherical
+        set arg1              = 8
+        set arg2              = 8
+    end
+end
+
+# --------------------------------------------------
+# Mesh Adaptation Control
+#---------------------------------------------------
+subsection mesh adaptation
+  set type                    = kelly
+  set variable                = pressure
+  set fraction type           = number
+  set max number elements     = 70000
+  set max refinement level    = 3
+  set min refinement level    = 1
+  set frequency               = 1
+  set fraction refinement     = 0.02
+  set fraction coarsening     = 0.01
+end
+
+# --------------------------------------------------
+# Non-Linear Solver Control
+#---------------------------------------------------
+subsection non-linear solver
+  set verbosity               = verbose
+  set tolerance               = 1e-4
```

Comment:
Can we decrease the tolerance?

Reply:
Done! I adjusted also the `max iterations` and the `minimum residual` :) I still have to update the figures when the simulation with the new tolerances will be over.